x-pack/plugin/esql/qa/testFixtures/src/main/resources/enrich.csv-spec

```diff
@@ -651,7 +651,6 @@ fieldsInOtherIndicesBug
 required_capability: enrich_load
 required_capability: fix_replace_missing_field_with_null_duplicate_name_id_in_layout
 required_capability: dense_vector_agg_metric_double_if_fns
-required_capability: exponential_histogram
```

Comment:
This is not needed anymore, because in `EsqlSpecTestCase` we already ensure that we only load the histogram-CSV data into an index if the capability is present.
Same for `lookup-join.csv-spec`